docs/source/configuration_file.md

```diff
@@ -0,0 +1,159 @@
+# Configuration file
+
+## General parameters
+
+#### command
+Run the specified command. Choices: ``"train"``, ``"test"``, ``"eval"``, to train, test and evaluate a model respectively.
+
+#### gpu
+Integer. ID of the GPU to use.
+
+#### log_directory
+Folder name that will contain the output files (e.g., trained model, predictions, results).
+
+#### debugging
+Bool. Extended verbosity and intermediate outputs.
+
+## Loader parameters
+
+#### bids_path
+String. Path of the BIDS folder.
+
+#### target_suffix
+Suffix list of the derivative file containing the ground-truth of interest (e.g. [`"_seg-manual"`, `"_lesion-manual"`]). If the list has a length greater than 1, then a multi-class model will be trained.
+
+#### contrasts
+- `train_validation`: list of image contrasts (e.g. `T1w`, `T2w`) loaded for the training and validation. If `multichannel`, then this list represent the different channels of the input tensors. Otherwise, the contrasts are mixed and the model has only one input channel.
+- `test`: list of image contrasts (e.g. `T1w`, `T2w`) loaded in the testing dataset. Same comment than for `train_validation` regarding `multichannel`.
+- `balance`: Dict. Enables to weight the importance of specific channels (or contrasts) in the dataset: e.g. `{"T1w": 0.1}` means that only 10% of the available `T1w` images will be included into the training/validation/test set. Please set `multichannel` to `false` if you are using this parameter.
+
+#### multichannel
+Bool. Indicated if more than a contrast (e.g. `T1w` and `T2w`) is used by the model. See details in both `train_validation` and `test` for the contrasts that are input.
+
+#### slice_axis
+Choice between `"sagittal"`, `"coronal"`, and `"axial"`.
+Sets the slice orientation for on which the model will be used.
+
+#### slice_filter
+Dict. Discard a slice from the dataset if it meets a condition, see below.
+- `filter_empty_input`: Bool. Discard slices where all voxel intensities are zeros.
+- `filter_empty_mask`: Bool. Discard slices where all voxel labels are zeros.
+
+#### roi
+Dict. of parameters about the region of interest
+- `suffix`: String. Suffix of the derivative file containing the ROI used to crop (e.g. `"_seg-manual"`) with `ROICrop` as transform. Please use `null` if you do not want to use a ROI to crop.
+- `slice_filter_roi`: int. It filters (i.e. discards from the dataset) slices where the ROI mask has with less than this number of non zero voxels. Active when using `"ROICrop"`.
+
+## Split dataset
+
+#### fname_split
+Filename of a joblib file containing the list of training/validation/testing subjects. This file can later be used to re-train a model using the same data splitting scheme.
+
+#### random_seed
+Seed used by the random number generator to split the dataset between training/validation/testing.
```

Comment:
what is expected? float? string? any range?

Reply:
Int is prefered, cf [here](https://docs.python.org/3/library/random.html#random.seed).
I am correcting this.